### PR TITLE
fix: dashboard /user/login get error code 405

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues
#396 
___
### Bugfix
- Description

- How to fix?

1. according to https://gist.github.com/zxhfighter/7560812  add `error_page  405  =200 $uri;`

1. in dashboard docker container, there is no hosts config about 'manage', so change

```
location /apisix/admin {
        proxy_pass http://manage:8080/apisix/admin;
    }
```
to
```
location /apisix/admin {
        proxy_pass http://192.17.5.15:8080/apisix/admin;
    }
```

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: compose/dashboard_conf/nginx.conf [hunk 1]
```diff
@@ -16,7 +16,8 @@ server {
     }
 
     location /apisix/admin {
-        proxy_pass http://manager:8080/apisix/admin;
+        proxy_pass http://192.17.5.15:8080/apisix/admin;
```
Comment (juzhiyuan) [inline compose/dashboard_conf/nginx.conf:19]: I don't think it's a good idea to use a fixed IP here, @gxthrj @bzp2010 what do you think?

**Comment by giphoo** [comment]
这里与是否ip无关，请参考修改：
```nginx
    location /user/login {
        if ( $request_method = 'POST' ) { #HERE
            proxy_pass http://manager:8080;
        }
        try_files $uri $uri/ /index.html;
    }
    location / {
        try_files $uri $uri/ /index.html;
    }

    location /apisix/admin {
        proxy_pass http://manager:8080/apisix/admin;
    }
```

**Review by moonming — APPROVED** [review]
`error_page` LGTM, but do we need the fixed IP?

**Review by juzhiyuan — APPROVED** [review]
LGTM

File: compose/dashboard_conf/nginx.conf (hunk 1 above)
Comment (gxthrj) [reply compose/dashboard_conf/nginx.conf]: It should not be changed here, not a problem here

**Pull request merged**